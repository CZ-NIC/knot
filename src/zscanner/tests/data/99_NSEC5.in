$ORIGIN	.
$TTL	1

; OK
@	NSEC5		65535	255	CPNMUOJ1E8====== TYPE0 TYPE32768
@	NSEC5KEY	255	Zm9vYg==
@	NSEC5PROOF	65535	Zm9vYg==
