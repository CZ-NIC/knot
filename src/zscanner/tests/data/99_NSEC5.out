OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=FF00
RDATA=FFFFFF06666F6F626172000180800180
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=FF01
RDATA=FF666F6F62
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=FF02
RDATA=FFFF666F6F62
------
